.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
           - --quiet
         files: ^(pytradfri|examples|tests)/.+\.py$
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 3.9.2
```

Comment:
Downgrade?

Reply:
Quite odd. Let's see how this tool behaves going forward. I'm using the Renovate tool for work and it also has support for Pre-commit, and has its own bot like dependabot. Might be better.

I'll update the branch here when I'm near a computer.